README.md

```diff
@@ -1 +1,35 @@
 # Ed-Fi-Standard
+
+**The Ed-Fi Data Standard serves as the foundation for enabling interoperability among secure education data systems.**
```

Comment:
Let's remove the bold formatting. I had it in the template to highlight what needed to be changed, and now I realize that you had no way of knowing that I didn't want to keep it.